haiku-apps/bescreencapture/bescreencapture-2.1.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="A screen recorder utility"
+DESCRIPTION="BeScreenCapture is a screen recorder utility for Haiku.
+It allows you to record what happens on your screen, then save it \
+to any media format supported in Haiku.
+BeScreenCapture can record either the entire screen, or just a section you \
+select."
+
+SUMMARY_inputfilter="Shortcut handler for BeScreenCapture"
+DESCRIPTION_inputfilter="Input Server Addon for BeScreenCapture. Allows the \
+user to launch BeScreenCapture and start/stop recording using a keyboard \
+combination (CTRL-COMMAND-SHIFT + R)."
+
+HOMEPAGE="https://github.com/jackburton79/bescreencapture"
+COPYRIGHT="2014-2016 Stefano Ceccherini"
+LICENSE="BSD (3-clause)
+	MIT
+	"
+REVISION="1"
+SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v2.1.tar.gz"
```

Comment:
This is not required, but it can be handy to use the **`$portVersion`** variable for **`SOURCE_URI`**:
```
SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v${portVersion}.tar.gz"
```